interface/main/tabs/menu/menus/standard.json

```diff
@@ -434,7 +434,7 @@
         ]
```

Comment:
It looks like you reverted this standard.json script to a prior version. Are there any changes in here pertinent to this PR?

Reply:
This branch was built off of  tywrenn:bootstrap-updates, thinking it would have bs4.  If you look at the code changes, I still had to bring in BS4 and popper so that is one area that needs to conform with the codebase - where ever/however bs4 is being brought in.
From the json menu, I changed the terms Rules and Alerts to Care Plans and Clinical Alerts. They point to a different urls since the front end has changed significantly.  These are all things you will need to look at and decide if you want to go that route.  I think what was once a complete mystery (CDR Engine) will start to make sense to the average admin/user.  The front end changes and some of the terminology in this PR simply highlight the CDR, a very powerful but significantly under appreciated module in the codebase.

Reply:
hi @ophthal ,
Doesn't look like your branch is based on tywrenn:bootstrap-updates now, but I can make that happen. Just let me know when you are in a gap in ongoing development, and I'll clean this up a bit and base it on the tywrenn:bootstrap-updates. Then you can take that new branch and post a new PR with it and work from that.
-brady